### ISSUE TITLE
Blog admin won't pull static assets from CDN

### ISSUE THREAD

**Issue by flowersjg** [johnnyflowers/schoolcalendars#6]
Makes the admin interface buggy.

**Comment by johnnyflowers** [comment]
If I understand this issue correctly, I should have a solution. Right now, our static file URLs point at http://d39ospbwcjyrg5.cloudfront.net/, which Zinnia won't load because it's not from the schoolcalendars.org domain. I've already got Namecheap and Cloudfront set up to use cdn.schoolcalendars.org as an alias to our Cloudfront/S3 bucket. Just need to test and get it pushed to prod. Bumping to high priority.

**Comment by johnnyflowers** [comment]
Using cdn.schoolcalendars.org doesn't quite work because the relevant static files are sending requests to www.schoolcalendars.org, and apparently browsers consider different subdomains to be cross-domain. I'll have to find a way to set document.domain = 'schoolcalendars.org' both in the django admin and in the tiny_mce static files. I've spent way too much time on this already, so downgrading to medium priority, and I'll revisit once I've got bigger things taken care of.
